sophia/src/term/iri.rs

```diff
@@ -36,11 +36,19 @@ pub enum Normalization {
 ///  - provides some identical methods to what `&str` provides (see below);
 ///  - can otherwise be converted to a `String` with `to_string`;
 ///
-/// See [module documentation](index.html)
-/// for more detail.
+/// # Contract
+///
+/// Each `Iri` represents a valid IRI according to the
```

Comment:
```suggestion
/// Each `Iri` represents a valid IRI references according to the
```

According to the RFC, an IRI is always absolute; an *IRI reference*
can be either absolute or relative.

Reply:
Note that I'm not suggesting that we change *all* mentions of IRI into "IRI references". That would be overly verbose,  even though strictly-speaking, most of the time we write "IRI" it actually means "IRI reference"...

But at least here, were we quote the RFC, I think we should use the proper term.